Service/Json/JsonReader.php

```diff
@@ -66,18 +66,26 @@ class JsonReader implements \Countable, \Iterator
     private $convertDocuments;
 
     /**
+     * @var array
+     */
+    private $options;
+
+    /**
      * Constructor.
      *
      * @param Manager $manager
      * @param string  $filename
+     * @param array   $options
      * @param bool    $convertDocuments
+     *
      */
-    public function __construct($manager, $filename, $convertDocuments = true)
+    public function __construct($manager, $filename, $options, $convertDocuments = true)
```

Comment:
I think the options here is the options of the command. Thus, ```$convertDocuments``` should not be put into the options.